tools/checklocks/test/atomics.go

```diff
@@ -27,6 +27,8 @@ type atomicStruct struct {
 
 	// +checklocksignore
 	ignored int32
+
+	wrapper1 atomic.Int32 // safe without any annotations
```

Comment:
nit: s/wrapper1/wrapper/

Reply:
done, squashed the commit into the previous one